packages/front-end/src/app/components/EGLabDetailsForm.vue

```diff
@@ -456,7 +456,7 @@
         :size="ButtonSizeEnum.enum.sm"
         type="submit"
         label="Edit"
-        :disabled="!useUserStore().canEditLab(useUserStore().currentOrgId)"
+        :disabled="useUserStore().isSuperuser || !useUserStore().canEditLab(useUserStore().currentOrgId)"
```

Comment:
`!useUserStore().canEditLab(useUserStore().currentOrgId)` might be a bit more readable as: 

`const hasEditPermission = !useUserStore().canEditLab(useUserStore().currentOrgId);`